cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinExternalSourcesBiobank.java

```diff
@@ -0,0 +1,28 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CristinExternalSourcesBiobank {
```

Comment:
https://github.com/BIBSYSDEV/nva-cristin-service/blob/main/cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/CristinExternalSource.java

Reply:
You can either move that one to commons, or, if you feel there is better to have a duplicate in local module, maybe we should keep it as is

Reply:
Why keep duplicates? If it is already in commons, use it.

Reply:
It is not in commons. I will move it there.